core/src/epicli/data/common/ansible/playbooks/roles/haproxy_runc/tasks/install-packages-RedHat.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install containerd.io package for RedHat family
```

Comment:
Is it the correct place to install packages? Should it be in 'repository' role?

Reply:
Same as before.